test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
@AuthorProxy I could, but three things:
* The project owner might not want to have that file in their project.
* I have to manually do this for every project I fork
* Wouldn't it be nice to just have an option to look for the first indentation in a file and have the linter use that as the validateIndentation setting?

I have decided that this PR and my initial thoughts of how this would work are incorrect. If I ever have time, I'll make a PR for an option to check for the first indentation in a file and then error on inconsistent indentations thereafter - this is what I really wanted in the first place I think.